docs/logs/getting-started-aspnetcore/README.md

```diff
@@ -0,0 +1,73 @@
+# Getting Started with OpenTelemetry .NET Logs in 5 Minutes - ASP.NET Core Application
+
+First, download and install the [.NET
+SDK](https://dotnet.microsoft.com/download) on your computer.
+
+Create a new web application:
+
+```sh
+dotnet new web -o aspnetcoreapp
+cd aspnetcoreapp
+```
+
+Install the
+[OpenTelemetry.Exporter.Console](../../../src/OpenTelemetry.Exporter.Console/README.md)
+and
+[OpenTelemetry.Extensions.Hosting](../../../src/OpenTelemetry.Extensions.Hosting/README.md)
+packages:
+
+```sh
+dotnet add package OpenTelemetry.Exporter.Console
+dotnet add package OpenTelemetry.Extensions.Hosting
+```
+
+Copy the [HelloLogs.cs](./HelloLogs.cs) and [Program.cs](./Program.cs) files to
+the project folder.
+
+Run the application again (using `dotnet run`) and then browse to the URL shown
+in the console for your application (e.g. `http://localhost:5154`). You should see
+the logs output from the console.
+
+```text
+LogRecord.Timestamp:               2023-09-01T22:55:50.6389757Z
+LogRecord.TraceId:                 8fa0ec5519c9bd5a498978c089a78182
+LogRecord.SpanId:                  b888feebe74e9685
+LogRecord.TraceFlags:              None
+LogRecord.CategoryName:            Program
+LogRecord.Severity:                Info
+LogRecord.SeverityText:            Information
+LogRecord.Body:                    Hello, world!
+LogRecord.Attributes (Key:Value):
+    OriginalFormat (a.k.a Body): Hello, world!
+LogRecord.EventId:                 1
+LogRecord.EventName:               SayHello
+LogRecord.ScopeValues (Key:Value):
+[Scope.0]:SpanId: b888feebe74e9685
```

Comment:
lets put an explanation here saying this scope is coming from asp.net core itself. and warn that some of the fields (like TraceId etc) are duplicate.

Separate:
(Its a good feature to provide feature to control scopes like option to limit the number of scope level/depth and which direction. A lot of users end up with all scopes from asp.net core itself, though their goal was to merely add a common userid/transactionid to whole logs themselves...)